packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx

```diff
@@ -1,23 +1,31 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { boolean } from '@storybook/addon-knobs';
 import { SeriesColorPicker, ColorPicker } from './ColorPicker';
 import { action } from '@storybook/addon-actions';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 import { UseState } from '../../utils/storybook/UseState';
 import { renderComponentWithTheme } from '../../utils/storybook/withTheme';
+import mdx from './ColorPicker.mdx';
 
 const getColorPickerKnobs = () => {
   return {
     enableNamedColors: boolean('Enable named colors', false),
   };
 };
 
-const ColorPickerStories = storiesOf('Pickers and Editors/ColorPicker/Pickers', module);
-
-ColorPickerStories.addDecorator(withCenteredStory);
+export default {
+  title: 'Pickers and Editors/ColorPicker',
+  component: ColorPicker,
+  subcomponents: { SeriesColorPicker },
+  decorators: [withCenteredStory],
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
 
-ColorPickerStories.add('default', () => {
+export const simple = () => {
```

Comment:
I wish we could call this `default` :( Maybe `basic` would fit better than simple? Something can be simple but not basic at the same time :)

Reply:
Yeah, basic is probably the way to go at the moment